bigtable/admin.go

```diff
@@ -1607,13 +1711,14 @@ func (it *BackupIterator) Next() (*BackupInfo, error) {
 
 // BackupInfo contains backup metadata. This struct is read-only.
 type BackupInfo struct {
-	Name        string
-	SourceTable string
-	SizeBytes   int64
-	StartTime   time.Time
-	EndTime     time.Time
-	ExpireTime  time.Time
-	State       string
+	Name           string
+	SourceTable    string
+	SizeBytes      int64
+	StartTime      time.Time
+	EndTime        time.Time
+	ExpireTime     time.Time
+	State          string
+	EncryptionInfo EncryptionInfo
```

Comment:
If this is nullable I would make the field a pointer so that nil can be distinguished from an empty struct. Then, you can make newEncryptionInfo also return a pointer (and make that pointer nil if the value passed in is nil), removing the need for the logic on L1680 and following.